README.rst

```diff
@@ -149,6 +149,28 @@ Advanced Usage
 
 See below for more options.
 
+Order Your Grouped Results with order_by and limit
+''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Alpha support for ordering your results and limiting what is displayed is now supported in the Python SDK.
+Keep in mind that even if you limit your results with the 'limit' keyword, you are still querying over the
+normal amount of data, and thus your compute costs will not change. Limit only changes what is displayed.
+
+The keyword 'limit' must be a positive integer. The keyword 'order_by' must be a dictionary with a required
```

Comment:
Aim for consistency with " and ' quotes. 

Tbh I wish we used the inline code stuff in this whole doc, alas probably something I'll do one day when I want to clean it up a bit...

Reply:
kk